Announcement for haskell summer of code 2023 (#167)
haskell-org/summer-of-haskell: content/faq.md

# Faq

Our policies are largely similar to that of Google Summer of Code, please refer
to their FAQ page found [here](https://developers.google.com/open-source/gsoc/faq).

> Some notable exceptions to Googles policy are:

Reviewer: ```suggestion
Some notable exceptions to Google's policies are:
```

>  -  Contributors can apply for more than one project

Reviewer: ```suggestion
 -  Contributors to Summer of Haskell can apply for up to two projects
```

>  -  We are following a slightly different timeline (see [homepage](./))

Reviewer: ```suggestion
 -  Summer of Haskell follows a slightly different timeline (see [homepage](./))
```
